src/features/keyboard/Keyboard.js

```diff
@@ -12,20 +12,20 @@ function Keyboard() {
 
         <div className='line'>
             {keys1.map((key)=>{
-                return <Key keyValue={key} />;
+                return <Key key={key} keyValue={key} />;
```

Comment:
Consider renaming some of the arrays/fields so the phrase "key" does not appear 10 times within 3 lines of code. :D